app/components/Exchange/MyMarkets.jsx

```diff
@@ -64,6 +65,35 @@ class MarketGroup extends React.Component {
         );
     }
 
+    _storeMarketChange(id) {
+        let {marketStats} = this.props;
+        let {marketsCache} = this.state;
+        
+        let timestamp = new Date().getTime();
+        let change = utils.format_number(marketStats.get(id) && marketStats.get(id).change ? marketStats.get(id).change : 0, 2);
+
+        if(!marketsCache[id]) { 
+            marketsCache[id] = [timestamp, "0.00", false];
+        }
+        
+        if(change != marketsCache[id][1] && marketsCache[id][1] != "0.00") { 
```

Comment:
Parsing as float below means you can use `!== 0` here. In general you should always use strict equalities.